### PR TITLE
#1 Changes to Signin and Signup pages

### PR DESCRIPTION
**Pull request by newCodeWriter**
1. Change material ui component for map function.
2. Add styling to the component.
3. Remove lowercase on values in travel list.

### CODE REVIEW

File: client/src/pages/SigninContainer.js [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 import React, { useReducer, useEffect } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
-import { TextField, Button } from "@material-ui/core";
+import { TextField, Button, IconButton } from "@material-ui/core";
```
Comment (johnsciutto) [inline client/src/pages/SigninContainer.js]: These two import statements are pulling from the same place (`@material-ui/core`).

File: client/src/pages/SignupOne.js [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 import React, { useReducer, useEffect } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
-import { TextField, Button } from "@material-ui/core";
+import { TextField, Button, IconButton } from "@material-ui/core";
```
Comment (johnsciutto) [inline client/src/pages/SignupOne.js]: Same as above.

File: client/src/pages/SignupTwo.js [hunk 3]
```diff
@@ -3,15 +3,13 @@
 import React, { useState } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
 import { useStyles } from "../styles/Signup_in";
-import { Button, Input } from "@material-ui/core";
+import { Button, Input, InputBase } from "@material-ui/core";
```
Comment (johnsciutto) [inline client/src/pages/SignupTwo.js]: Same as above.

**Review by sina-jamshidi — APPROVED** [review]
LGTM!

**Pull request merged**